### PR TITLE
Commercial containers - IE fix

### PR DESCRIPTION
**Pull request by Calanthe**
## What does this change?

Continuation of the https://github.com/guardian/frontend/pull/14207. Markup changes in `itemLargeCard.scala.html` and `itemSmallCard.scala.html`. Also fixes for the IE 11. There is still an issue with the wrong markup of badges. This will be done later.

## Screenshots

Before IE11:
image_ 21

After IE11:
screen shot 2016-09-19 at 15 52 00

On IE9 it doesn't look perfect but every element is visible:
screen shot 2016-09-19 at 17 20 42

screen shot 2016-09-19 at 17 20 57

There is a lot of js errors in IE9 and I keep getting new ones every time I open the inspect tool:
screen shot 2016-09-19 at 17 20 24

Imo it's not worth polishing IE9 any more.
## Request for comment

@regiskuckaertz

### CODE REVIEW

File: common/app/views/commercial/cards/itemLargeCard.scala.html [hunk 1]
```diff
@@ -13,30 +13,32 @@
...
-    </div>
-</a>
+            @if(useCardBranding) {
+                @for(branding <- item.branding) {
+                    @contentLogo(branding)
+                }
+            }
+        </div>
+    </a>
+        <div class="advert__image-container @for(imgRatio <- optImgRatio){u-responsive-ratio u-responsive-ratio--@imgRatio}">
```
Comment (regiskuckaertz) [inline common/app/views/commercial/cards/itemLargeCard.scala.html]: The image is not clickable then?

File: common/app/views/commercial/cards/itemCard.scala.html [hunk 2]
```diff
@@ -16,19 +16,19 @@
 <div class="@CssClassBuilder.linkFromStandardCard(item, optAdvertClassNames, optClassNames, useCardBranding)">
     <a href="@item.targetUrl"
        data-link-name="@omnitureId"
-       class="advert">
+       class="advert__container">
```
Comment (regiskuckaertz) [inline common/app/views/commercial/cards/itemCard.scala.html]: I thought `.advert__container` would be used for the surrounding `DIV` 😅  The idea is that everything that happens inside an `.advert` (its layout and sizes and stuff) does not need to apply to the surrounding `DIV`.

Comment (regiskuckaertz) [inline common/app/views/commercial/cards/itemCard.scala.html]: In fact, I think the whole IE11 bug is caused by a selector in [_adverts.scss line 204](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/commercial/_adverts.scss#L204) that I wrote 😭  Instead, it should be: `.advert__row .advert__image-container`, not `.advert__row > * > .advert__image-container`

**Review by regiskuckaertz — CHANGES_REQUESTED** [review]
I think these changes demonstrate some pitfalls in the CSS I originally wrote (sorry 😞 ), in particular that I used wrong combinations of direct child selectors where, like the .advert__image-container example above. I'll pull your branch and try to fix that.

File: common/app/views/commercial/cards/itemLargeCard.scala.html (hunk 1 above)
Comment (Calanthe) [inline common/app/views/commercial/cards/itemLargeCard.scala.html]: Yes! It should be clickable.

**Comment by Calanthe** [comment]
@regiskuckaertz Are you going to continue working on this issue?

**Comment by regiskuckaertz** [comment]
@Calanthe yes! I may be able to fix this quicker.

**Comment by regiskuckaertz** [comment]
Reviving this PR @Calanthe 

Before:
picture 3

After:
picture 4

**Comment by regiskuckaertz** [comment]
(logos are not showing up because some weird vibe with browserstack)

**Comment by regiskuckaertz** [comment]
@lps88 mind reviewing this one I took from Zofia?

**Review by regiskuckaertz — APPROVED** [review]

**Comment by lps88** [comment]
👍  LGTM

**Comment by regiskuckaertz** [comment]
Nice nice nice! Thanks @lps88

**Pull request merged**